### PR TITLE
adds spherical brush

### PR DESCRIPTION
**Pull request by jsutker**

### CODE REVIEW

File: src/cursor.cpp [hunk 1]
```diff
@@ -4,6 +4,10 @@ int Cursor::getSize () {
   return size;
```
Comment (jovanduy) [inline src/cursor.cpp:4]: should this return `radius` (not `size`)?

File: src/main.cpp [hunk 2]
```diff
@@ -125,6 +125,18 @@ void handleInput (unsigned char key, int x, int y) {
       break;
     case 'h':
       displayUsage();
+      break;
+    case '1':
+      cursor->setShape(1);
```
Comment (jovanduy) [inline src/main.cpp]: in your comment for get/set shape, you note that the cursor shape options are 0 and 1, but here it seems like they are 1 and 2?

Comment (jovanduy) [reply src/main.cpp]: please add these commands to the help list (either under "other" or change "move cursor" to "cursor commands" and put them there)

**Review by jovanduy — CHANGES_REQUESTED** [review]
Currently, this doesn't compile. I tried messing with some of the places I commented on to get it to compile, but there was no cursor-reshape functionality.

File: src/cursor.cpp (hunk 1 above)
Comment (jsutker) [reply src/cursor.cpp:4]: there were a few discrepancies, whoops. fixed now

File: src/main.cpp (hunk 2 above)
Comment (jsutker) [reply src/main.cpp]: the shape values were off by 1 for the keys and the encoded shapes, I changed where that shows up to make it more clear

**Comment by jovanduy** [comment]
Awesome, it compiles now! 👍 However, maybe I'm missing something, but everything is still just a cube. When I toggle between 1 and 2 (and when I try placing in both modes) nothing changes... Maybe it's just an issue with my computer, but it also didn't work on my Ubuntu VM.

**Review by jovanduy — APPROVED** [review]

**Pull request merged**